packages/reporter/cypress/integration/unit/events_spec.ts

```diff
@@ -363,16 +363,13 @@ describe('events', () => {
       expect(runner.emit).to.have.been.calledWith('runner:unpin:snapshot', 'command id')
     })
 
-    it('emits focus:tests on focus:tests', () => {
-      events.emit('focus:tests')
-      expect(runner.emit).to.have.been.calledWith('focus:tests')
-    })
-
```

Comment:
This test change is pretty representative of what's going on in this PR - we no longer "focus the launchpad that's showing the specs list" but instead, change it as a setting that's part of reporter state, just like `autoScrollingEnabled` has worked until now.